cmake-toolchain/README.md

```diff
@@ -0,0 +1,40 @@
+# Cross Compiliation
+ **Notice:** Running the AWS IoT Device Client will incur usage of AWS IoT services, and is likely to incur charges on your AWS account. Please refer the pricing pages for [AWS IoT Core](https://aws.amazon.com/iot-core/pricing/), [AWS IoT Device Management](https://aws.amazon.com/iot-device-management/pricing/), and [AWS IoT Device Defender](https://aws.amazon.com/iot-device-defender/pricing/) for more details.
+
+[*Back To The Main Readme*](../README.md)
+
+## Building from one architecture to the other
+
+**Description**:
+Cross Compiling allows you to compile for a **specific architecture** while on a **different architecture**.  This is extremely useful, as some *(Especially really constrained)* devices won't have the resources to complete the build process.  Even if your devices can complete the build, it is often times much easier to build in a central location (like your developer laptop) and then distribute the artifacts to appropriate devices. 
+
+The `cmake-toolchain` folder contains toolchains that will make cross compiling for other architectures easier. Currently we have toolchains to support cross compiling for the following architectures; **MIPS32**, **ARMhf**, & **AArch64**.
```

Comment:
**MIPS32**, **ARMhf**, & **AArch64**.  I think we should be more professional here with wording and avoid using '&'. Suggest to change to: 
**MIPS32**, **ARMhf**,  and **AArch64**

Reply:
The only thing they would be required to download is whatever specific compiler is required

Reply:
In that case, I don't think that qualifies as providing the toolchain since the toolchain is basically the compiler, assembler, debugger, linker, ...etc (other tools you need to produce the executable), the things that we don't provide and require the users to download themselves.

Reply:
I changed it to call them tool chain files, and added a note that the compiler is not included in this